packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -103,9 +103,9 @@ describe("Runtime", () => {
                                         state: "disabled",
                                     },
                                 },
+                                flushMode,
```

Comment:
We should probably add tests to this file that validates container runtime correctly adds `batch` metadata and generates batch events in both flush modes.